packages/editor/__tests__/specs/toggleRequiredLabel.test.ts

```diff
@@ -0,0 +1,19 @@
+import {toggleRequiredLabel} from '../../src/client/toggleRequiredLabel'
```

Comment:
I'm curious about testing: is it necessary to write a test for such a small thing ? Why do you think it's necessary ?

Reply:
Mistakes happen, this just makes sure they don't in the future. Especially when refactoring things like this makes it easier.

If functions are small and pure usually writing tests are quick. I use snapshots for these kind of tests to use as regression testing, as if we change the symbol then we can just press `u` to update the snapshots easily.

There's things I agree that do not need to have unit tests (getters or setters for example), but anywhere there is logic involved, tests should be present.